### DISCUSSION TITLE
Shortcuts control

### DISCUSSION THREAD

**Discussion by kouyoumin** [gopro/labs#643] [Feature Requests]
There's a "SHORTCUTS" section in the profile setting, which assign functions to screen buttons or mode button.
I used to assign speed ramp to mode button in timewarp mode so I can toggle real speed using the remote.
(It's impractical to touch the screen button when camera is neck or chest mounted.)

In the latest H12 firmware, speed ramp finally comes to max timewarp.
However, it can only be toggled by screen button.
There's no mode button setting in max timewarp profile.
Can this be done via labs QR control?
(I hope the button setting is still there but just hidden by the GUI.)

**Comment by dnewman-gpsw** [comment]
There isn't a way to do this through Labs, although I asked internally, as that does feel correct.

**Reply by kouyoumin** [reply]
Can I look forward to the future release (no matter labs or not) for this?
